### PR TITLE
Data norm chris

### PR DESCRIPTION
**Pull request by cgwhitehead**
I wasn't super sure how best to proceed. Since the project should stay relatively small in the future (ie we don't have a huge team to build custom integrations with different city systems), I thought it best to have one style csv we use for bulk adding and then various pipelines as appropriate to convert into the right format. 

Then we can use the framework you started to feed the clean data (possibly reformatted to json or whatever is needed) into the db. Or we can bulk load csv's which it looks like mongo import can handle. 

Let me know what you think or if you see anything that can be improved or standards that could be better followed or anything that could make it better. 

putting the clean csv in google drive as a sample.

### CODE REVIEW

File: data-normalization/pipeline.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    Takes any number of fields
+    return list of attachements or other feature
+    or list ['nothing yet']
+    """
+    attached=[]
+    for x in args:
+        if x != None:
+            attached.append(x)
+    if len(attached)==0:
+        return ['nothing yet']
```
Comment (MatthewScholefield) [inline data-normalization/pipeline.py]: To me this seems like it'd make more sense to return just an empty list (`[]`). What do you think?

**Review by MatthewScholefield — COMMENTED** [review]
This is awesome! I tried it out and it worked nicely, producing some really clean data.

File: data-normalization/pipeline.py (hunk 1 above)
Comment (cgwhitehead) [reply data-normalization/pipeline.py]: Yeah, that makes sense. I'll make the change. Thanks!

**Review by cgwhitehead — COMMENTED** [review]
Looks great. Do I need to approve b4 a merge?

**Review by cgwhitehead — COMMENTED** [review]
check

**Review by cgwhitehead — COMMENTED** [review]
Probably an obvious question but what does the * do?

**Comment by MatthewScholefield** [comment]
@cgwhitehead The `*` expands the list into a set of arguments:
```python
def foo(a, b):
    print('a:', a, 'b:', b)

my_list = [1, 2]
foo(my_list, 3)  # a: [1, 2] b: 3

def bar(a, b, c):
    print('a:', a, 'b:', b, 'c:', c)

my_list = [1, 2]
bar(*my_list, 3)  # a: 1 b: 2 c: 3

def baz(*args):
    print('args:', args)

baz(1, 2, 3)  # args: [1, 2, 3]
baz(*my_list, 3)  # args: [1, 2, 3]
```

**Comment by MatthewScholefield** [comment]
Also, I forgot to comment but I made those changes and left them for a bit to see if they looked okay to you. Since it sounds like it looks good, I'll merge.

**Pull request merged**